src/main/scala/cz/alenkacz/db/postgresscala/DbValue.scala

```diff
@@ -0,0 +1,6 @@
+package cz.alenkacz.db.postgresscala
+
+class DbValue(value: Any) {
```

Comment:
not sure I understand. You are proposing the interface to have methods like string() and the fact that it is casted from String can be different for different drivers?